### DISCUSSION TITLE
[A-L] (2023/24) Foglio 3 - Esercizio 1

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#180] [Esercizi - Viaggi/Piazza] [answered]
![Screenshot 2023-10-22 204347](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/49d3b192-3c43-4d4f-a58f-e3c2c8fcd7b4)

**Comment by Elia-Belli** [comment, marked as answer]
Utilizziamo l'algoritmo di Euclide:
$a=14322 = 6153 \cdot 2 + 2016$
$b=6153 = 2016 \cdot 3 +105$
$r_1=2016= 105 \cdot 19 + 21$
$r_2=105 = 21 \cdot 5 + 0$
$\Rightarrow MCD(a,b)=21$

$r_1 = 2016= a+(-2)\cdot b =(1,0)+(0,1)(-2)=(1,-2)$
$r_2= 105 = b + 2016 \cdot (-3)= (0,1)+(1,-2)(-3) = (-3,7)$
$r_3 = 21 = 2016+105\cdot (-19)= (1,-2)+(-3,7)(-19)=(1,-2)+(57,-133)=(58,-135)$
**Identità di Bezout :** $r_3 = a(58)+b(-135)$

**Comment by daromacarlo** [comment]
![Immagine 2023-12-12 103617](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/967813e1-ef95-467e-9cdc-da4976e1b6ad)

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/d457c4ea-a2e3-4b5e-b117-659e3c755c87)
